doc/getting_started.rst

```diff
@@ -305,6 +309,9 @@ Now we can connect to the serial console:
 
     $ labgrid-client -p example-place console
 
+.. note:: Using remote connection requires ``microcom`` installed on the host
+   where the labgrid-exporter is running.
```

Comment:
This is incorrect. `microcom` is executed on the same host as `labgrid-client`. `ser2net` is executed on the host running the exporter.

Reply:
Thank! I only tested on one host (both exporter and client) so this might be the source of my mistake. I'll try to setup a 'really' remote environment and try again.